doc/book/in-depth-guide/first-module.md

```diff
@@ -158,11 +160,11 @@ return [
             // Define a new route called "blog"
             'blog' => [
                 // Define a "literal" route type:
-                'type' => 'literal',
+                'type' => Literal::class,
                 // Configure the route itself
                 'options' => [
                     // Listen to "/blog" as uri:
-                    'route'    => '/blog',
+                    'route' => '/blog',
```

Comment:
again, it is inconsistent, for example why above `type` and `options` are not aligned to the same column?